src/test/java/com/ericsson/ei/frontend/RegExProviderUtilsTest.java

```diff
@@ -0,0 +1,19 @@
+package com.ericsson.ei.frontend;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import com.ericsson.ei.frontend.utils.RegExProviderUtils;
+
+public class RegExProviderUtilsTest {
+
+    @Test
+    public void test() {
+        String regExForInvalidName = RegExProviderUtils.getRegEx("invalidName");
+        String regExForValidEmail = RegExProviderUtils.getRegEx("validEmail");
+        assertEquals(regExForInvalidName, "(\\W)");
```

Comment:
Please remove the regexps here and instead validate that it is not empty, since if regexp changes then we have to change here.
so assertFalse(fetchedRegexp.isEmpty())

Reply:
done